compiler/src/iree/compiler/Dialect/VMVX/vmvx.imports.mlir

```diff
@@ -47,6 +47,31 @@ vm.import @add.2d.f32(
   %sizes : tuple<i64, i64>
 )
 
+//==============================================================================
+// Strided copy ops
+// Variants of copy ops exist for power of two rank and datatype sizes.
+// Current max rank is 2d.
+//==============================================================================
+vm.import @copy.2d.x8(
```

Comment:
Could you have one and pass in the element-size instead? That would help contain the combinatorial explosion if >2d is added.

It's the same with fill: a fill that took an i64 + a value counting the number of bytes in the pattern would remove the x32/etc variants. We do this in the HAL today for similar reasons (one command buffer fill op can cover all of the valid 1/2/4/8 widths).

Your dialect could take the real type and the lowering into the import could do the getElementBitWidth (or w/e) and vm.ext... the operand to the full width.